specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2019-05-06-preview/searchindex.json

```diff
@@ -0,0 +1,1336 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SearchIndexClient",
+    "description": "Client that can be used to query an index and upload, merge, or delete documents.",
+    "version": "2019-05-06-Preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "syncMethods": "None"
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "https://{searchServiceName}.{searchDnsSuffix}/indexes('{indexName}')",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/SearchServiceNameParameter"
+      },
+      {
+        "$ref": "#/parameters/SearchDnsSuffixParameter"
+      },
+      {
+        "$ref": "#/parameters/IndexNameParameter"
+      }
+    ]
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/docs/$count": {
+      "get": {
+        "tags": [
+          "Documents"
+        ],
+        "operationId": "Documents_Count",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/Count-Documents"
+        },
+        "x-ms-examples": {
+          "SearchIndexCountDocuments": {
+            "$ref": "./examples/SearchIndexCountDocuments.json"
+          }
+        },
+        "description": "Queries the number of documents in the Azure Search index.",
```

Comment:
Remove "Azure Search". Should be "Queries the number of documents in the index."